tests/func-tests/wasp_test.go

```diff
@@ -0,0 +1,98 @@
+package tests_test
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	tests "github.com/kubevirt/hyperconverged-cluster-operator/tests/func-tests"
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/tests/flags"
+)
+
+const (
+	setWaspFGPatchTemplate = `[{"op": "replace", "path": "/spec/featureGates/enableHigherDensityWithSwap", "value": %t}]`
+)
+
+var _ = Describe("wasp-agent", Label("wasp"), Serial, Ordered, func() {
```

Comment:
Please add the new "OpenShift" label. See #2860 
```suggestion
var _ = Describe("wasp-agent", Label("wasp", "OpenShift"), Serial, Ordered, func() {
```

Reply:
@nunnatsa Done